src/jit/emit.cpp

```diff
@@ -2969,9 +2999,10 @@ emitter::instrDesc  * emitter::emitNewInstrCallInd(int        argCnt,
                                                    VARSET_VALARG_TP GCvars,
                                                    regMaskTP  gcrefRegs,
                                                    regMaskTP  byrefRegs,
-                                                   emitAttr   retSizeIn)
+                                                   emitAttr   retSizeIn
+                                                   FEATURE_MULTIREG_RET_ONLY_ARG(emitAttr secondRetSizeIn))
 {
-    emitAttr  retSize = retSizeIn ? EA_ATTR(retSizeIn) : EA_PTRSIZE;
+    emitAttr       retSize = (retSizeIn != EA_UNKNOWN) ? retSizeIn : EA_PTRSIZE;
 
     bool gcRefRegsInScratch = ((gcrefRegs & RBM_CALLEE_TRASH) != 0);
```

Comment:
Yes, the comment above needs to include the need for a LargeCall in the case where the RETURN is a gc-type (not just when gc types are live at the point of call). It is too easy (witness my confusion) to confuse the need to report gc types that are in registers at the point of call, with the need to have a LargeCall for completely different reasons.